### ISSUE TITLE
JSON.parse error in getTargets

### ISSUE THREAD

**Issue by Jesseyx** [RemoteDebug/remotedebug-ios-webkit-adapter#57]
I installed this package with npm. There is log

```bash
iosAdapter.getTargets
remotedebug-ios-webkit-adapter is listening on port 9000
undefined:1
<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
^

SyntaxError: Unexpected token < in JSON at position 0
    at Object.parse (native)
    at Request.request [as _callback] (/usr/local/lib/node_modules/remotedebug-ios-webkit-adapter/out/adapters/iosAdapter.js:44:38)
    at Request.self.callback (/usr/local/lib/node_modules/remotedebug-ios-webkit-adapter/node_modules/request/request.js:188:22)
    at emitTwo (events.js:106:13)
    at Request.emit (events.js:191:7)
    at Request.<anonymous> (/usr/local/lib/node_modules/remotedebug-ios-webkit-adapter/node_modules/request/request.js:1171:10)
    at emitOne (events.js:96:13)
    at Request.emit (events.js:188:7)
    at IncomingMessage.<anonymous> (/usr/local/lib/node_modules/remotedebug-ios-webkit-adapter/node_modules/request/request.js:1091:12)
    at IncomingMessage.g (events.js:292:16)
```

**Comment by auchenberg** [comment]
Looks like the adapter is fetching HTML on port 9222/9223. Do you have a
local web server running on those ports?

**Comment by auchenberg** [comment]
Ping @Jesseyx

**Comment by Jesseyx** [comment]
Because i used a global agent. Thanks!

**Comment by menglexiang** [comment]
I also encountered this problem

➜  ~ lsof -i:9000                  
➜  ~ lsof -i:9221                  
➜  ~ lsof -i:9222                  
➜  ~ lsof -i:9223                  
➜  ~ remotedebug_ios_webkit_adapter
iosAdapter.getTargets
undefined:1
<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
^

SyntaxError: Unexpected token < in JSON at position 0
    at JSON.parse (<anonymous>)
    at Request.request [as _callback] (/usr/local/lib/node_modules/@tencent/remotedebug-ios-webkit-adapter/out/adapters/iosAdapter.js:44:38)
    at Request.self.callback (/usr/local/lib/node_modules/@tencent/remotedebug-ios-webkit-adapter/node_modules/request/request.js:186:22)
    at emitTwo (events.js:126:13)
    at Request.emit (events.js:214:7)
    at Request.<anonymous> (/usr/local/lib/node_modules/@tencent/remotedebug-ios-webkit-adapter/node_modules/request/request.js:1163:10)
    at emitOne (events.js:116:13)
    at Request.emit (events.js:211:7)
    at IncomingMessage.<anonymous> (/usr/local/lib/node_modules/@tencent/remotedebug-ios-webkit-adapter/node_modules/request/request.js:1085:12)
    at Object.onceWrapper (events.js:313:30)

**Comment by NicolasVargas** [comment]
Hi, I solved it by adding "127.0.0.1" to NO_PROXY environment variable

**Comment by hsxfjames** [comment]
Same as @NicolasVargas mentioned, but I solved it by unsetting `http_proxy` and `https_proxy` env var.
